api/config/config.go

```diff
@@ -162,6 +162,15 @@ type StandardTransformerConfig struct {
 	FeastCoreURL          string `envconfig:"FEAST_CORE_URL" required:"true"`
 	FeastCoreAuthAudience string `envconfig:"FEAST_CORE_AUTH_AUDIENCE" required:"true"`
 	EnableAuth            bool   `envconfig:"FEAST_AUTH_ENABLED" default:"false"`
+	Jaeger                JaegerConfig
+}
+
+type JaegerConfig struct {
+	AgentHost    string `envconfig:"JAEGER_AGENT_HOST"`
+	AgentPort    string `envconfig:"JAEGER_AGENT_PORT"`
+	SamplerType  string `envconfig:"JAEGER_SAMPLER_TYPE" default:"const"`
+	SamplerParam string `envconfig:"JAEGER_SAMPLER_PARAM" default:"1"`
+	Disabled     string `envconfig:"JAEGER_DISABLED" default:"false"`
```

Comment:
If the tracing enabled by default, we should change the jaeger sampler to `probabilistic` with param value to be `0.1` (or less). Having `const` as jaeger sampler type would use a lot of space and also affect the transformer performance in production.

Reply:
I prefer to have 0.01 as default.